[2023-11-20T08:43:29.958+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T08:43:29.998+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T08:43:30.001+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:43:30.020+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-05 00:00:00+00:00
[2023-11-20T08:43:30.037+0000] {standard_task_runner.py:57} INFO - Started process 180 to run task
[2023-11-20T08:43:30.053+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-05T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpunw187ei']
[2023-11-20T08:43:30.071+0000] {standard_task_runner.py:85} INFO - Job 37: Subtask load_data
[2023-11-20T08:43:30.134+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [running]> on host c84b35a13244
[2023-11-20T08:43:30.220+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-05T00:00:00+00:00'
[2023-11-20T08:43:30.864+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Scripts/etl.py", line 179, in load_data
    engine = create_engine(DATABASE_URL)
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/deprecations.py", line 375, in warned
    return fn(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 514, in create_engine
    u = _url.make_url(url)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 738, in make_url
    return _parse_url(name_or_url)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 794, in _parse_url
    components["port"] = int(components["port"])
ValueError: invalid literal for int() with base 10: '{}'
[2023-11-20T08:43:30.869+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231105T000000, start_date=20231120T084329, end_date=20231120T084330
[2023-11-20T08:43:30.873+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 37 for task load_data (invalid literal for int() with base 10: '{}'; 180)
[2023-11-20T08:43:30.907+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-20T08:43:30.916+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:46:45.571+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T08:46:45.575+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T08:46:45.575+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:46:45.580+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-05 00:00:00+00:00
[2023-11-20T08:46:45.583+0000] {standard_task_runner.py:57} INFO - Started process 201 to run task
[2023-11-20T08:46:45.586+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-05T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmph0148osw']
[2023-11-20T08:46:45.588+0000] {standard_task_runner.py:85} INFO - Job 48: Subtask load_data
[2023-11-20T08:46:45.618+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [running]> on host d26cbef9e8ab
[2023-11-20T08:46:45.654+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-05T00:00:00+00:00'
[2023-11-20T08:46:48.388+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:46:48.393+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231105T000000, start_date=20231120T084645, end_date=20231120T084648
[2023-11-20T08:46:48.418+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:46:48.427+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:50:23.722+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T08:50:23.743+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T08:50:23.746+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:50:23.768+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-05 00:00:00+00:00
[2023-11-20T08:50:23.772+0000] {standard_task_runner.py:57} INFO - Started process 173 to run task
[2023-11-20T08:50:23.780+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-05T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp41mk2nvi']
[2023-11-20T08:50:23.786+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask load_data
[2023-11-20T08:50:23.868+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [running]> on host 3ef39241d6e6
[2023-11-20T08:50:24.022+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-05T00:00:00+00:00'
[2023-11-20T08:50:27.237+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:50:27.248+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231105T000000, start_date=20231120T085023, end_date=20231120T085027
[2023-11-20T08:50:27.293+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:50:27.307+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T22:32:40.037+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T22:32:40.056+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T22:32:40.058+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T22:32:40.118+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-05 00:00:00+00:00
[2023-11-20T22:32:40.144+0000] {standard_task_runner.py:57} INFO - Started process 208 to run task
[2023-11-20T22:32:40.171+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-05T00:00:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp7fbhlexp']
[2023-11-20T22:32:40.218+0000] {standard_task_runner.py:85} INFO - Job 113: Subtask load_data
[2023-11-20T22:32:40.437+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [running]> on host ad6c889005f6
[2023-11-20T22:32:40.732+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-05T00:00:00+00:00'
[2023-11-20T22:32:43.500+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T22:32:43.543+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231105T000000, start_date=20231120T223240, end_date=20231120T223243
[2023-11-20T22:32:43.606+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T22:32:43.669+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T22:47:44.775+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T22:47:44.801+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [queued]>
[2023-11-20T22:47:44.810+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T22:47:44.849+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-05 00:00:00+00:00
[2023-11-20T22:47:44.864+0000] {standard_task_runner.py:57} INFO - Started process 145 to run task
[2023-11-20T22:47:44.871+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-05T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpdu7_7hy7']
[2023-11-20T22:47:44.879+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask load_data
[2023-11-20T22:47:44.971+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-05T00:00:00+00:00 [running]> on host fda681cc78c9
[2023-11-20T22:47:45.111+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-05T00:00:00+00:00'
[2023-11-20T22:47:50.171+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T22:47:50.220+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231105T000000, start_date=20231120T224744, end_date=20231120T224750
[2023-11-20T22:47:50.353+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T22:47:50.426+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
